src/main/java/org/sunbird/workflow/config/Configuration.java

```diff
@@ -804,4 +807,12 @@ public List<String> getBulkUploadCategoryValue() {
     public void setBulkUploadCategoryValue(String bulkUploadCategoryValue) {
         this.bulkUploadCategoryValue = bulkUploadCategoryValue;
     }
+
+    public String getGroupValues() {
+        return groupValues;
```

Comment:
Just return this as List<String> -- refer the cb-ext implementation for the same for how comma separated String is returned as String.

Reply:
This has been implemented as suggested.